### PR TITLE
Improve geometry_to_bing_tiles error message 

### LINKED ISSUE #11599: Improve The number of input tiles is too large (more than 1M) error message
Improve `The number of input tiles is too large (more than 1M) to compute a set of covering Bing tiles.` error message in `geometry_to_bing_tiles` by including the following information:

- the rectangle that cannot be covered;
- zoom level;
- number of tiles that's exceeding the limit

CC: @jagill

### PR DESCRIPTION
**Pull request by picrinite**
Fixes #11599 
1. Improve error message for geo-function geometry_to_bing_tiles with zoom level,  tile number and rectangle geometry in wkt format 
2. Improve error message for converting circle to bing tiles
with zoom level,  tile number and radius in error message 
cc: @mbasmanova

### CODE REVIEW

File: presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/BingTileFunctions.java [hunk 1]
```diff
@@ -427,10 +428,11 @@ private static BingTile getTileCoveringLowerRightCorner(Envelope envelope, int z
         return tile;
     }
 
-    private static void checkGeometryToBingTilesLimits(OGCGeometry ogcGeometry, boolean pointOrRectangle, long tileCount)
+    private static void checkGeometryToBingTilesLimits(OGCGeometry ogcGeometry, boolean pointOrRectangle, long tileCount, int zoomLevel)
     {
         if (pointOrRectangle) {
-            checkCondition(tileCount <= 1_000_000, "The number of input tiles is too large (more than 1M) to compute a set of covering Bing tiles.");
+            checkCondition(tileCount <= 1_000_000, "%d input tiles is too large (more than 1M) to compute a set of " +
```
Comment (mbasmanova) [inline presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/BingTileFunctions.java]: perhaps, `The number of tiles covering input rectangle exceeds the limit of 1M. Number of tiles: %s. Rectangle: xMin=%s,  yMin=%s, xMax=%s, yMax=%s. Zoom level: %s.`

This way, we have a fixed prefix which allows for searching the code and details at the end of the message to provide some clues to the user.

**Comment by picrinite** [comment]
Updated error message format

File: presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestBingTileFunctions.java [hunk 2]
```diff
@@ -239,7 +239,7 @@ public void testBingTilesWithRadiusBadInput()
 
         // Too many tiles
         assertInvalidFunction("bing_tiles_around(30.12, 60.0, 20, 100)",
-                "The number of input tiles is too large (more than 1M) to compute a set of covering Bing tiles.");
+                "The number of tiles covering input rectangle exceeds the limit of 1M. Number of tiles: 36699364. Radius_km: 100.0. Zoom level: 20.");
```
Comment (mbasmanova) [inline presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestBingTileFunctions.java]: Let's use *more proper* English for `Radius_km: 100.0`: `Radius: 100.0 km.`

**Comment by mbasmanova** [comment]
@picrinite Vic, the code changes look good. The commit message needs some editing. How about this?

```
Improve error messages in geometry_to_bing_tiles and bing_tiles_around

Include zoom level, number of tiles and rectangle in the `number of tiles too large` error message.
```

**Pull request merged**